homeassistant/components/madvr/remote.py

```diff
@@ -0,0 +1,85 @@
+"""Support for MadVR remote control."""
+
+from collections.abc import Iterable
+import logging
+from typing import Any
+
+from homeassistant.components.remote import RemoteEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import MadVRConfigEntry
+from .const import DOMAIN
+from .coordinator import MadVRCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: MadVRConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the MadVR remote."""
+    coordinator = entry.runtime_data
+    async_add_entities(
+        [
+            MadvrRemote(hass, coordinator, entry.entry_id),
+        ]
+    )
+
+
+class MadvrRemote(CoordinatorEntity[MadVRCoordinator], RemoteEntity):
+    """Remote entity for the MadVR integration."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        coordinator: MadVRCoordinator,
+        entry_id: str,
+    ) -> None:
+        """Initialize the remote entity."""
+        super().__init__(coordinator)
+        self.madvr_client = coordinator.client
+        self._attr_unique_id = coordinator.mac
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, f"{self.coordinator.mac}")},
```

Comment:
```suggestion
            identifiers={(DOMAIN, coordinator.mac)},
```
Also consider adding the mac as connections. I don't have the right way to do it at me because im on a different PC, but I will come back to this if you haven;t found it in the codebase yet. (just search for connections)

Reply:
added connections